Api Web/Models/Department.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Api_Web.Models
+{
+    public class Department
+    {
+        [Key]
```

Comment:
If we add only the Id, we don't need the attribute key

Reply:
Changes has been done, please check. @HeyBaldur